src/viur/core/bones/uri.py

```diff
@@ -0,0 +1,204 @@
+import fnmatch
+import sys
+
+import logging
+from . import BaseBone, ReadFromClientError, ReadFromClientErrorSeverity
+from urllib.parse import urlparse, urlunparse
+import typing as t
+from collections.abc import Iterable
+from collections import namedtuple
```

Comment:
`logging` and `sys` are not used!
```suggestion
import fnmatch
import typing as t
from . import BaseBone, ReadFromClientError, ReadFromClientErrorSeverity
from urllib.parse import urlparse, urlunparse
from collections.abc import Iterable
from collections import namedtuple
```